mifospay/src/main/java/org/mifos/mobilewallet/mifospay/kyc/ui/KYCDescriptionFragment.java

```diff
@@ -37,23 +41,41 @@
 
     KYCContract.KYCDescriptionPresenter mKYCDescriptionPresenter;
 
-    @BindView(R.id.cv_lvl1)
-    CardView cvLvl1;
+    @BindView(R.id.lllv1)
+    LinearLayout lllv1;
 
-    @BindView(R.id.cv_lvl2)
-    CardView cvLvl2;
+    @BindView(R.id.lllv2)
+    LinearLayout lllv2;
 
-    @BindView(R.id.cv_lvl3)
-    CardView cvLvl3;
+    @BindView(R.id.lllv3)
+    LinearLayout lllv3;
 
-    @BindView(R.id.ll_lvl1)
-    LinearLayout llLvl1;
+    @BindView(R.id.blv1)
+    Button blv1;
 
-    @BindView(R.id.ll_lvl2)
-    LinearLayout llLvl2;
+    @BindView(R.id.blv2)
+    Button blv2;
 
-    @BindView(R.id.ll_lvl3)
-    LinearLayout llLvl3;
+    @BindView(R.id.blv3)
+    Button blv3;
+
+    @BindView(R.id.completedtx1)
+    TextView txt1;
```

Comment:
Keep these as same as the id. Like for this one you would change it to `completedtx1`.